services/account/controllers/AccountController.ts

```diff
@@ -12,27 +12,15 @@ export class AccountController {
   }
 
   async getAccount(filter) {
-    try {
```

Comment:
Currently the filter object is coming in as a string. It's converted to an object still but as Raymond pointed out [here](https://github.com/strongloop/loopback-next-example/pull/61#pullrequestreview-46096320) the promise will be rejected if there is an error and hence we don't need an explicit rejection using try / catch.